forui/lib/src/widgets/label.dart

```diff
@@ -47,6 +47,7 @@ enum FLabelState {
 /// * [FLabelStyles] for customizing a label's appearance.
 final class FLabel extends StatelessWidget {
   /// The label's style. Defaults to the appropriate style in [FThemeData.labelStyles].
+  // ignore: diagnostic_describe_all_properties
```

Comment:
Curious, why do we remove the diagnostic here?

Reply:
We ignore the diagnostic because the field is a record which causes the widget inspector to crash.